src/screens/FeedScreen/index.js

```diff
@@ -79,7 +81,9 @@ const FeedScreen = (props) => {
 
   React.useEffect(() => {
     const unsubscribe = navigation.addListener('focus', () => {
-      showSearchBarAnimation();
+      if (!route.params.isGoBack) {
+        showSearchBarAnimation();
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces a condition to check if `route.params.isGoBack` is not true before executing `showSearchBarAnimation()`. This seems like a logical change, but it's important to ensure that the `isGoBack` parameter is always available and correctly set in the route params. If it's not guaranteed, consider adding a fallback value.



<!-- This is an auto-generated comment by OSS CodeRabbit -->